### PR TITLE
Fix upgrade test

### PR DESCRIPTION
**Pull request by nunnatsa**
Make sure the "replaces" in v100.0.0 SCV, matches to the base version deployed.

Signed-off-by: Nahshon Unna-Tsameret <nunnatsa@redhat.com>

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

File: hack/upgrade-create-new-version.sh [hunk 1]
```diff
@@ -33,7 +40,13 @@ cp -r ${LATEST_CSV_DIR} ${UPGRADE_CSV_DIR}
...
 sed -i "s|${OPERATOR_NAME}.v${LATEST_VERSION}|${OPERATOR_NAME}.v${UPGRADE_VERSION}|g" "${UPGRADE_CSV}"
-sed -i "s|replaces:.*|replaces: ${OPERATOR_NAME}.v${LATEST_VERSION}|" "${UPGRADE_CSV}"
+sed -i "s|replaces:.*|replaces: ${OPERATOR_NAME}.v${REPLACES_VERSION}|" "${UPGRADE_CSV}"
 sed -i "s|version:\s*${LATEST_VERSION}|version: ${UPGRADE_VERSION}|g" "${UPGRADE_CSV}"
 sed -i "s|value:\s*${LATEST_VERSION}|value: ${UPGRADE_VERSION}|g" "${UPGRADE_CSV}"
 sed -i "/^channels:/a - name: \"${UPGRADE_VERSION}\"\n  currentCSV: ${OPERATOR_NAME}.v${UPGRADE_VERSION}" ./deploy/olm-catalog/kubevirt-hyperconverged/kubevirt-hyperconverged.package.yaml
+
+if [ "${RELEASE_DELTA}" != "1" ]; then
+  echo "  skips:" >> "${UPGRADE_CSV}"
+  echo "  - ${LATEST_VERSION}" >> "${UPGRADE_CSV}"
```
Comment (tiraboschi) [inline hack/upgrade-create-new-version.sh:50]: if RELEASE_DELTA is > 2 we need two lines and so on...

**Comment by nunnatsa** [comment]
/hold

Still not working. waiting for #716

**Pull request closed without merging**